src/lib/utilities/get-login-url.ts

```diff
@@ -1,6 +1,7 @@
 export const getLoginUrl = (
   settings: Settings,
   currentSearchParams: URLSearchParams,
+  redirect_url: string,
```

Comment:
Nit: Why does this have an underscore? The convention is camelcase.

Reply:
Also: Should just be optional with a `?` so that we don't have to pass `null` in constantly?

You could just do `redirectUrl = null` and get the same effect without _having_ to always passing `null` in.

Reply:
after debugging various cases changed the logic that redirectUrl is now expected to come with `currentSearchParams: URLSearchParams`